### DISCUSSION TITLE
Curation on PubPub

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#1416] [Ideas]
Hello! I'm wondering if folks have thought much about what they'd like to see in terms of curation, for lack of a better word, on PubPub. What I mean by this is creating Pubs (or some other object) that mainly exists to link to another object elsewhere on the web, potentially with some additional context.

I know some folks are already doing interesting work there, like:

- [Rapid Reviews](https://rapidreviewscovid19.mitpress.mit.edu/emerging-variants-topic)
- [crimrxiv](https://www.crimrxiv.com/pub/pcrcao82/release/1)
- I think @jeffpooley, too, although I can't find a specific example

We've thought about making it much, much easier to create Connections, probably via a dialogue when you add a link to the Pub body, or maybe even a way to create a simple 'curation Pub' from anywhere. We've also thought about a connection type that would essentially be a passthrough similar to the way Reddit links are — that is, in Pub Blocks it would probably link directly to the external resource, but still have a link into the Pub itself for comments/additional context.

I'm also interested to know all the ways folks are thinking about other forms of curation/duplication/republishing and the general idea of a Pub that's mostly meant to refer to another digital object, and if you have ideas for how we could better support it.

**Comment by gabestein** [comment]
From crimrxiv: https://twitter.com/CriminologyOpen/status/1394754426519314436

**Reply by sanjeetsinghpr** [reply]
I have to head out for an appointment, but I think related material will be a good fit.

**Comment by writingpoli** [comment]
I think I mentioned this elsewhere -- something like fyi.to. See our community resources page: sareview.org/resources for one place we'd use it. 

I wouldn't mind the ability to incorporate RSS feeds of other items from around the Web, too.

**Comment by jeffpooley** [comment]
Thanks for asking about this. One way we've been using it is to republish, in overlay-journal fashion, public domain or openly licensed materials, using the Version Connection, e.g. [here](https://www.mediastudies.press/pub/james-self/).

In this case, it's part of an "open reader" meant for undergraduate courses—so the [whole Collection](https://www.mediastudies.press/sms) is an act of curation, except for the intro and preface.

Just a side note on that project, because it sounds compatible with the passthrough Connection type idea you floated, Gabe. We have, in this "open reader" concept, the idea of outbound links, for articles that are on the free Internet but not licensed for republication. Here's an [example](https://www.mediastudies.press/pub/tolentino-internet/release/4). For those we also use a Version-style Connection, and add some other sign-posts, along with two paragraphs of fair-use excerpting. 

For our [forthcoming journal](https://hms.mediastudies.press), we'd like to include overlay republication of OA articles among the mix, and imagine using the Connection (Version) for that too. 

As for the MediArXiv preprint server I'm also involved in, we are seriously considering switching to PubPub as our platform, in the [CrimRxiv](https://www.crimrxiv.com/) mold. Either way, we intend to set up a PubPub community that would allow for Publish, Review, Curate (PRC) style review and commentary.

I've found the existing Connection interface to be pretty clear, but I see how it could be hard to find. It's a great idea to make the Connection feature more visible, maybe even as a secondary page off of the main Create Pub button, with a radio button option for a "curation Pub" (not sure that title captures the concept, maybe "connected Pub"?), that would include brief explanatory text and a link to use cases? A drop-down off the Create Pub button could work too, but I have a feeling that a bit of carefully crafted explanation will go a long way on the wider uptake of this feature.

**Comment by writingpoli** [comment]
Regarding the existing Connections tool, we've used it to:

1. connect versions of pieces (usually when there was an error in importing more than one or wholesale changes were made): 
2. connect related pieces -- like, the review of the book by the author of this essay: https://www.sareview.org/pub/sar4hagermanwhitekids or the review of the book by this poet: https://www.sareview.org/pub/sar2salinasshark/release/2
3. and maybe one other use I'm not thinking of (is there any way to see all the existing connections for a journal?)

But, playing with it just now, I can certainly think of ways we could make more use of it -- if only by connecting related works by authors/genre/etc.

**Comment by writingpoli** [comment]
On this, possible to add "Resource" as an option for connection relation? 

![Screenshot_20210612-080427](https://user-images.githubusercontent.com/3883763/121776934-2e0a0780-cb55-11eb-86eb-b06aec005da1.png)

**Reply by gabestein** [reply]
Totally, and/but, I think we might try to keep the types at least loosely based on existing ontologies if possible. Right now all of them are either one of, or, in the case of rejoinder, aliased to a [Crossref relationship type](https://www.crossref.org/documentation/content-registration/structural-metadata/relationships/). I'll be the first to admit that that is not the richest or best ontology to link it to, but if one of those types works, or if it could be aliased to one of those types, it would help, as right now connections are pretty built into the deposit system. How do you feel about `Related material`?

Longer term, I am increasingly interested in the linked data taxonomies conversation happening in #1386, and would love to think of connections more broadly as how we start to add rich data to Pubs across a broad range of ontologies.

**Reply by writingpoli** [reply]
'Bout to run out to an appointment, but quickly, I think Related material will work. But I'm going to look into the Crossref types later -- does any auto-import of the info happen into Crossref? Pondering whether documenting existing websites and data repositories as resources helps/hinders their database/others' research efforts.

**Reply by gabestein** [reply]
We automatically deposit any connections as Relations to Crossref. So, theoretically, one could traverse those graphs. I think it's probably net-helpful, and most people I've talked to want more of this, not less. But I can certainly see downsides.

**Reply by writingpoli** [reply]
Can we also offer the option to edit the kind of connection it is after saving? I've had to delete and start over so many times.

**Comment by jflournoy** [comment]
We're looking into pubpub for putting out a journal that is essentially just a curation of existing literature (in trad journals or not) using the connections feature. At the moment, I don't like the idea of copy/pasting *any* content from the publications we're curating to avoid any perception of parasitism. At the moment, our plan is to use connections like "This pub is a *comment* on..." with a brief explanation of why we are including the publication in an issue. But it would be nice to be able to customize the look and feel of these "publications" to background our own identities somewhat. While it's nice to show which user curates which articles, it would be nice for the publication not to show up as being "By Joe Smoe" when all Joe Smoe has done is link to Sam Kelly's publication. I also wouldn't want to say that the curation publication is "by" Sam Kelly, because Sam hasn't consented.

Does this make sense? In other words, I think a lot might be accomplished through a little bit of aesthetic flexibility.

**Reply by eastofesten** [reply]
If I'm understanding your example, you could in PubPub Legacy remove "Joe Smoe" as a contributor (or not give "Joe" byline contribution) to hide how names appear on the pub itself. Similarly, you could hide "Sam Kelly" as a Contributor via the Connections feature. But it sounds like you want to pursue additional customization of the look & feel for the pubs and the journal in general, which is something we're thinking about.

**Reply by jflournoy** [reply]
We just published our first overlay issue (https://dsl.pubpub.org/volume-1-issue-1-july-2024) so I have a little bit more information about what I was really envisioning. Where we got to is that it would be useful for us to be able to mark connection as "This is a comment on..." but be able to place the connection at the bottom of the page. What we did is just set the connection to "A comment on this pub" even though that's not technically correct. But the meaning is clear. I think the authorship designation is actually fine as is, at least for our use case. :D

**Comment by saqi004** [comment]
It would be great to make curation on PubPub easier by allowing users to quickly create Pubs that link to external resources with added context and comments. Streamlining the process[,](https://menuland.ph/burger-king-menu-prices-in-philippines/) introducing connection types, and enabling direct passthrough links could enhance the user experience. Integrating RSS feeds or APIs could automate updates, while tags or filters would help with organization and discovery.
